starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans:300,600");
+
+body {
+    font-family: "Open Sans",Helvetica,Arial,sans-serif;
+    font-size:11pt;
+}
+
+.recipeTitle {
+    margin: auto;
+    display: block;
+    text-align: center;
+    font-size: 70pt;
+    font-weight: normal;
+    font-family:Helvetica, Arial;
+    color: #ffffff;
+    }
+
+#myTitle {
+    background:url("../images/apple-pie.jpg");
+    background-size: cover;
+    padding-top:350px;
+    padding-bottom:350px;
```

Comment:
That's not really a good practice to use `padding` to set the height of an element, it would have been better to use a `<div>` containing the `<h1>` and to set the `height: 700px;` for the `<div>` and to set the position of the `<h1>` using `margin`.

But your code works so that's okay !